util/linuxfw/nftables_runner.go

```diff
@@ -804,6 +804,32 @@ func (n *nftablesRunner) AddChains() error {
 	return n.conn.Flush()
 }
 
+// createDummyPostroutingChains is used to create dummy postrouting chains in
+// netfilter via nftables, as a last resort measure to detect that nftables can
+// be used. It cleans up the dummy chains after creation.
+func (n *nftablesRunner) createDummyPostroutingChains() error {
+	const chainName = "ts-test-postrouting"
+	polAccept := nftables.ChainPolicyAccept
+	for _, table := range n.getNATTables() {
+		// Create the nat table if it doesn't exist, this table name is the same
+		// as the name used by iptables-nft and ufw. We install rules into the
+		// same conventional table so that `accept` verdicts from our jump
+		// chains are conclusive.
```

Comment:
why does the table name matter if this is a dummy chain for testing whether we can make any chain at all?

Reply:
detection works with a `ts-test-nat` naem, so did that